packages/kbn-alerts-ui-shared/src/common/hooks/use_alerts_data_view.test.tsx

```diff
@@ -20,14 +20,15 @@ import { testQueryClientConfig } from '../test_utils/test_query_client_config';
 import { useAlertsDataView } from './use_alerts_data_view';
 
 jest.mock('../apis/fetch_alerts_index_names');
-const mockFetchAlertsIndexNames = jest
-  .mocked(fetchAlertsIndexNames)
-  .mockResolvedValue([
+const mockFetchAlertsIndexNames = jest.mocked(fetchAlertsIndexNames).mockResolvedValue({
+  indexName: [
     '.alerts-observability.uptime.alerts-*',
     '.alerts-observability.metrics.alerts-*',
     '.alerts-observability.logs.alerts-*',
     '.alerts-observability.apm.alerts-*',
-  ]);
+  ],
+  hasReadIndexPrivilege: false,
```

Comment:
If we are going to use this field at the end, could you please add some tests for `hasReadIndexPrivilege: true` behaviour?